src/NetMQ/Core/SocketBase.cs

```diff
@@ -993,7 +995,7 @@ public bool TrySend(ref Msg msg, TimeSpan timeout, bool more)
         /// </remarks>
         /// <exception cref="FaultException">the Msg must already have been uninitialised</exception>
         /// <exception cref="TerminatingException">The socket must not already be stopped.</exception>
```

Comment:
Would be good to document `OperationCanceledException` here too, and a few other places in this file.

Reply:
It is not throwing, I only throw from non-Try methods. The Try methods return false.